WordPress-Extra/ruleset.xml

```diff
@@ -33,6 +33,7 @@
 	<rule ref="WordPress.WP.PreparedSQL"/>
 	<rule ref="WordPress.Variables.GlobalVariables"/>
 	<rule ref="WordPress.PHP.StrictComparisons" />
+	<rule ref="WordPress.WP.I18n" />
```

Comment:
Should this go into `WordPress-Extra` or `WordPress-Core`? I'm just concerned about use with _actual_ core, and how a lot of warnings will start appearing since no text domain is supplied.